### PR TITLE
feat(fwc): add landing warnings

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #[issue_no]

## Summary of Changes

Add two reminder warnings during landing:
- GND SPLR NOT ARMED
- SPD BRK STILL OUT

## Screenshots (if necessary)

image
image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

- GND SPLR NOT ARMED
  - Should occur during approach below 800 ft RA when gear is down, speedbrakes and retracted, and ground spoilers are not armed.
- SPD BRK STILL OUT
  - Should occur during climb (above 1500 feet)/cruise/descent (above 800 feet):
    - When an engine is above idle and the speedbrakes are deployed for more than 80 seconds,
    - During a TCAS RA requiring positive V/S if the speedbrakes are deployed,
  - Should occur during approach:
    - When speedbrakes are out below 800 feet RA.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by Eearslya — APPROVED** [review]

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #7863
Tier of Testing  : 1
Date             : 15/03/2023

Testing Process: 
GND SPLR NOT ARMED
Should occur during approach below 800 ft RA when gear is down, speedbrakes and retracted, and ground spoilers are not armed. - **OK**

SPD BRK STILL OUT
Should occur during climb (above 1500 feet)/cruise/descent (above 800 feet),
When an engine is above idle and the speedbrakes are deployed for more than 80 seconds, - **OK**
During a TCAS RA requiring positive V/S if the speedbrakes are deployed, - unable to test

Should occur during approach:
When speedbrakes are out below 800 feet RA. - **OK**

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

Media:       
image

File: fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts [hunk 1]
```diff
@@ -796,8 +852,10 @@ export class PseudoFWC {
         this.engine2State.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_STATE:2', 'Enum'));
         this.N1Eng1.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_N1:1', 'number'));
         this.N1Eng2.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_N1:2', 'number'));
-        this.N1IdleEng1.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_IDLE_N1:1', 'number'));
-        this.N1IdleEng2.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_IDLE_N1:2', 'number'));
+        this.N2Eng1.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_N2:1', 'number'));
+        this.N2Eng2.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_N2:2', 'number'));
+        this.N1IdleEng.set(SimVar.GetSimVarValue('L:A32NX_ENGINE_IDLE_N1', 'number'));
+        const oneEngineAboveMinPower = this.engine1AboveIdle.get() || this.engine2AboveIdle.get();
```
Comment (beheh) [inline fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts:858]: After havingg seen this on a stream just now: I think this is missing a comparison, like >= 20 (you gotta figure out a good N1 threshold somewhere).

Comment (tracernz) [reply fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts:858]: Yeah, I noticed that. The current comparison is using a threshold of 0.5 above the idle N1… but the engines actually seem to sit up to 1 above idle N1 in descent.

Comment (tracernz) [reply fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts:858]: Really the ECU should calculate this anyway.

**Pull request merged**